parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -139,11 +141,15 @@ function Retry-Command {
         $RetryDelaySeconds
     )
 
-    for ($i = 0; $i -lt $Retries; $i++) {
+    for ($i = 0; ; ) {

```

Comment:
where's the exit condition?

Reply:
It will return if success or throw an exception if it fails with max retries.